api/proto/teleport/userpreferences/v1/userpreferences.proto

```diff
@@ -40,6 +40,8 @@ message UserPreferences {
   UnifiedResourcePreferences unified_resource_preferences = 5;
   // access_graph is the preferences for Access Graph.
   AccessGraphUserPreferences access_graph = 6;
+  // keyboard_layout is the layout used by the user in Windows Desktop sessions
```

Comment:
could use a comment about what mapping from int32 --> keyboard layout this is using